packages/response-ops/rule_params/kibana.jsonc

```diff
@@ -1,5 +1,9 @@
 {
   "type": "shared-common",
   "id": "@kbn/response-ops-rule-params",
-  "owner": "@elastic/response-ops"
-}
+  "owner": [
+    "@elastic/response-ops"
+  ],
+  "group": "platform",
+  "visibility": "private"
```

Comment:
This package will grow to contain the schema and the types of all rule parameters from all rule types. Should we change it to `shared` as it will probably used by the solutions?

Reply:
I'd answer the same as in [this comment](https://github.com/elastic/kibana/pull/199630#discussion_r1847883722).
If you're certain that this will evolve to be used by all solutions (in terms of imported code) in the near future... then I'm okay changing it to `shared`.

Reply:
We can leave it as it is at the moment and change it when the time comes if it is something that can be easily done.

Reply:
Ok, let's leave this as `private` atm. Thanks for the explanation.